mrp_production_request/models/mrp_production_request.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+import odoo.addons.decimal_precision as dp
+from odoo.exceptions import UserError
+
+
+class MrpProductionRequest(models.Model):
+    _name = "mrp.production.request"
+    _description = "Manufacturing Request"
+    _inherit = "mail.thread"
+    _order = "date_planned_start desc, id desc"
+
+    @api.model
+    def _company_get(self):
+        company_id = self.env['res.company']._company_default_get(self._name)
+        return self.env['res.company'].browse(company_id.id)
+
+    @api.model
+    def _get_default_requested_by(self):
+        return self.env.user
+
+    name = fields.Char(
+        default="/", required=True,
+        readonly=True, states={'draft': [('readonly', False)]})
+    origin = fields.Char(
+        string='Source Document',
+        readonly=True, states={'draft': [('readonly', False)]})
+    requested_by = fields.Many2one(
+        comodel_name='res.users', string='Requested by',
+        default=_get_default_requested_by,
```

Comment:
To be overidable: Then _get_default_requested_by() method definition can be placed after field declaration

```suggestion
        default=lambda self: self._get_default_requested_by,
```